### PR TITLE
hls AES use available IV or 'Random.new()' from 'Crypto'

### LINKED ISSUE #835: [7908] Failed to execute script svtplay-dl
recent call last):
  File "svtplay-dl", line 7, in <module>
  File "site-packages\svtplay_dl-1.9.9-py3.5.egg\svtplay_dl\__init__.py", line 514, in main
  File "site-packages\svtplay_dl-1.9.9-py3.5.egg\svtplay_dl\__init__.py", line 217, in get_media
  File "site-packages\svtplay_dl-1.9.9-py3.5.egg\svtplay_dl\__init__.py", line 246, in get_all_episodes
  File "site-packages\svtplay_dl-1.9.9-py3.5.egg\svtplay_dl\__init__.py", line 340, in get_one_media
  File "site-packages\svtplay_dl-1.9.9-py3.5.egg\svtplay_dl\fetcher\hls.py", line 113, in download
  File "site-packages\svtplay_dl-1.9.9-py3.5.egg\svtplay_dl\fetcher\hls.py", line 175, in _download
  File "site-packages\Crypto\Cipher\blockalgo.py", line 295, in decrypt
ValueError: Input strings must be a multiple of 16 in length
[7908] Failed to execute script svtplay-dl

### PR DESCRIPTION
**Pull request by dalgr**
fixes #835

### CODE REVIEW

File: lib/svtplay_dl/fetcher/hls.py [hunk 1]
```diff
@@ -93,6 +94,7 @@ def hlsparse(options, res, url, **kwargs):
             streams[int(bit_rate)] = HLS(copy.copy(options), urls, bit_rate, cookies=res.cookies, keycookie=keycookie, authorization=authorization, audio=audio_url)
 
     elif m3u8.media_segment:
+        options.segments = False
```
Comment (spaam) [inline lib/svtplay_dl/fetcher/hls.py]: why do you set this?

Comment (dalgr) [reply lib/svtplay_dl/fetcher/hls.py]: It is an unrelated bug fix that makes it possible to download hls with only the url to the m3u8 media_segment when it only contains one segment. Ex:
```
./svtplay-dl -S -M -f "https://mtgxse04-vh.akamaihd.net/i/open/20xxxxxx/2049xxxx/204952xx/20495279/5a30e587ea29838b8cc56e8f7c82d233_20495279_269270_20180222T213434_,150,300,600,1200,2400,.mp4.csmil/master.m3u8?__b__=2000&hdnts=st=1519667985~exp=3667151632~acl=/*~hmac=2e33027cf399db818ca38ef80d645ef3dbde1ed1ac3f773f04000b3c26a48386&cc1=name=Svenska%20f%C3%B6r%20h%C3%B6rselskadade~default=no~forced=no~lang=sv~uri=https%3A%2F%2Fsubstitch.play.mtgx.tv%2Fsubtitle%2Fconvert%2Fvtt%3Fsource%3Dhttps%3A%2F%2Fcdn-subtitles-mtgx-tv.akamaized.net%2Fpitcher%2F20xxxxxx%2F2049xxxx%2F204952xx%2F20495279%2F2800120619%2Funified.subs.20495279_sv_sdh.vtt%26output%3Dm3u8"
```

Comment (dalgr) [reply lib/svtplay_dl/fetcher/hls.py]: Makes it possible to download m3u8 media_segments, Ex:
```
./svtplay-dl -S -M -f "https://mtgxse04-vh.akamaihd.net/i/open/20xxxxxx/2049xxxx/204952xx/20495279/5a30e587ea29838b8cc56e8f7c82d233_20495279_269270_20180222T213434_,150,300,600,1200,2400,.mp4.csmil/master.m3u8?__b__=2000&hdnts=st=1519667985~exp=3667151632~acl=/*~hmac=2e33027cf399db818ca38ef80d645ef3dbde1ed1ac3f773f04000b3c26a48386&cc1=name=Svenska%20f%C3%B6r%20h%C3%B6rselskadade~default=no~forced=no~lang=sv~uri=https%3A%2F%2Fsubstitch.play.mtgx.tv%2Fsubtitle%2Fconvert%2Fvtt%3Fsource%3Dhttps%3A%2F%2Fcdn-subtitles-mtgx-tv.akamaized.net%2Fpitcher%2F20xxxxxx%2F2049xxxx%2F204952xx%2F20495279%2F2800120619%2Funified.subs.20495279_sv_sdh.vtt%26output%3Dm3u8"
```

Comment (dalgr) [reply lib/svtplay_dl/fetcher/hls.py]: Before i complained that `options.segments` was not defined

**Pull request merged**